src/constants.ts

```diff
@@ -260,9 +262,7 @@ ExecStartPost=/usr/bin/touch /var/local/.custom-first-boot-done
 RemainAfterExit=yes
 
 [Install]
-WantedBy=basic.target
```

Comment:
Why did we have these and why do we not want them anymore?

Reply:
@ondrejbudai noticed these are in fact incorrect or have no effect and I can confirm that these are useless